mxnet/buildspec.yml

```diff
@@ -1,16 +1,16 @@
   account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
   region: &REGION <set-$REGION-in-environment>
   framework: &FRAMEWORK mxnet
-  version: &VERSION 1.7.0
+  version: &VERSION 1.6.0
   cuda_version: &CUDA_VERSION cu101
   os_version: &OS_VERSION ubuntu16.04
 
   repository_info:
-    training_repository: &TRAINING_REPOSITORY
-      image_type: &TRAINING_IMAGE_TYPE training
-      root: !join [ *FRAMEWORK, "/", *TRAINING_IMAGE_TYPE ]
-      repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *TRAINING_IMAGE_TYPE]
-      repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ] 
+    # training_repository: &TRAINING_REPOSITORY
```

Comment:
can we revert this back once this is ready for merge?

Reply:
reverted